cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,75 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import homePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import ProfilePageObject from '../support/pages/profile.PageObject';
+import faker from 'faker';
+
+const homePage = new homePageObject();
+const settingsPage = new SettingsPageObject();
+const profilePage = new ProfilePageObject();
+
+// the faker didn't work, so I hardcoded the test data. This is a screenshot of the faker error
+// https://prnt.sc/THXOTEbzSox4
+
+  const testData = {
+  // username: faker.name.firstName(),
+  // bio: faker.lorem.word(5),
+  // email: faker.internet.email(),
+  // password: faker.internet.password({ length: 8 })
+  username: 'edited_name',
+  bio: 'edited_bio',
+  email: 'edited_email@qa.team',
+  password: 'Edited_password123'
+  };
 
 describe('Settings page', () => {
+  let user;
   before(() => {
-
+    //I left this comment here because I have a question why this function is not working
```

Comment:
Remove the commented code, and find out why the cy.task('generateTestData') function is not working.

Reply:
ok =)